client/galaxy/scripts/components/show_params.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+    <div id="show-dataset-params" class="dataset-metadata" v-cloak>
+        <h2>
+            {{ toolName }}
+        </h2>
+        <h3>Dataset Information</h3>
+        <table class="tabletip" id="dataset-details">
+            <tbody>
+                <tr><td> Number:</td><td> {{ hdaData.number }} </td></tr>
+                <tr><td> Name:</td><td> {{ hdaData.name }} </td></tr>
```

Comment:
For other people new to Vue and reviewing this - I Googled it and Vue is automatically escaping things - very nice!